inbm/cloudadapter-agent/tests/unit/cloud/adapters/inbs/test_operation.py

```diff
@@ -107,13 +108,16 @@
             "<schedule_request>"
             "<request_id>1234</request_id>"
             "<update_schedule>"
-            "<schedule><single_schedule /></schedule>"
+            "<schedule><single_schedule>"
+            "<job_id>swupd-2c5d08c1-200a-49cf-b808-e53c74a22e86</job_id>"
```

Comment:
**<h2>$\color{#FFD700}Coding Style$</h2>**
**Suggestion:** Ensure consistent use of single quotes or double quotes for strings throughout the code to maintain consistency.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            '<job_id>swupd-2c5d08c1-200a-49cf-b808-e53c74a22e86</job_id>'
```